src/winetricks

```diff
@@ -718,6 +718,15 @@ w_expand_env()
     winetricks_early_wine cmd.exe /c echo "%$1%"
 }
 
+# Check if $1 is executable (output custom message using w_debug if $2 is specified)
+w_check_exec()
+{
+    if ! command -v "$1" >/dev/null; then
+        [ -z "$2" ] && { printf "Command '%s' is not executable\n" "$1" && exit 1 ;} || printf '%s\n' "$2"
```

Comment:
Please use an if statement here to make it more clear what's going on.

Reply:
Resolved in https://github.com/Winetricks/winetricks/pull/1351/commits/02be4e1b93647e0d3dd154573d7589b02a58f8db